.env.test

```diff
@@ -0,0 +1,4 @@
+DB_NAME=lunch_test
+SUPERUSER_NAME=test
+SUPERUSER_PASSWORD=test
+SUPERUSER_EMAIL=test@lunch.pink
```

Comment:
This file shouldn't be checked in.

Reply:
> It was written solely for use by a disposable container on CI, so it seemed
harmless. But if we're going to use the test env more generally, then yeah,
we'll have to generate it somehow at build time.

@svoynow For one thing, it's in `.gitignore`, so there's no guarantee that changes will be tracked. See `README.md` as it states that you should be creating `.env.test` yourself.